modules/FhirAPI/src/FhirAPI/FhirRestApiBuilder/Parts/Strategy/StrategyElement/ValueSet/FhirValueSetMapping.php

```diff
@@ -37,6 +37,9 @@ public function DBToFhir(...$params){
 
         // build expansion (contains the actual codes)
         if(in_array('$expand', $operations)) {
+
+            $this->
```

Comment:
oops..... fix it....

Reply:
I hope it is mistake and not something important , too long have Pass since this pull I can't recall